x-pack/platform/plugins/shared/alerting/server/routes/gaps/apis/find/transforms/transform_response/v1.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FindGapsResponseBodyV1 } from '../../../../../../../common/routes/gaps/apis/find';
+
+import { Gap } from '../../../../../../lib/rule_gaps/gap';
+
+export const transformResponse = ({
+  page,
+  perPage,
+  total,
+  data: gapsData,
+}: {
+  page: number;
+  perPage: number;
+  total: number;
+  data: Gap[];
+}): FindGapsResponseBodyV1 => ({
+  page,
+  per_page: perPage,
+  total,
+  data: gapsData
+    .map((gap) => {
+      if (!gap?.internalFields?._id || !gap.timestamp) {
+        return undefined;
+      }
+      return {
+        _id: gap.internalFields._id,
+        ...gap.getEsObject(),
```

Comment:
nit: rename `getEsObject` to more generic `toObject` since we're not just returning it for ES?

Reply:
https://github.com/elastic/kibana/commit/09e8d383351e358b251898c80e4c2a13c92454e2